### ISSUE TITLE
[Suggestion] Friendlier Board Member Page

### ISSUE THREAD

**Issue by williamkapke** [nodejs/foundation.nodejs.org#5]
This will be `CMS`d content that is controlled by... probably... marketing/PR. So, since I don't really know a conduit to communicating to those folks- I'm posting here....

I'd like to propose that, when this site goes live, there is a _friendlier_ Board Member Page.

The [current page](https://nodejs.org/en/foundation/board/) lists the info.. which is does its job... but I think we can help alleviate any perception of a secret corporate overload board (feedback I've heard) by simply putting faces to names. Certainly there isn't a requirement in the by-laws for someone to have a picture up to be on the board... but I bet many will be fine with it.

Picking a random example for illustration, [AirBnB's founders page](https://www.airbnb.com/about/founders):
![screenshot 2016-03-23 17 11 54](https://cloud.githubusercontent.com/assets/739813/14004877/c9aaa124-f11a-11e5-96f4-3437eb9ffd8c.png)

See! Nice, friendly faces :D

**Comment by rvagg** [comment]
good idea, many of the board members are professionals who are used to having their faces splashed around so on the most part I bet they wouldn't object, the objectors could opt for whatever avatar they are most comfortable with
